Boa.Constrictor/WebDriver/Extensions/AbstractWebPropertyListQuestion.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace Boa.Constrictor.WebDriver
+{
+    /// <summary>
+    /// Abstract class for any Web questions that return a collection of strings using a Web locator.
```

Comment:
/// Abstract class for any Web questions that return a collection of strings using a Web locator and a property name.